src/components/molecules/Menu.module.css

```diff
@@ -9,36 +9,89 @@
 .logo {
   white-space: nowrap;
   display: flex;
+  flex: 0 0 auto;
+  flex-direction: row;
+  justify-content: center;
   align-items: center;
-  order: 1;
 }
 
 .navigation {
-  order: 3;
-  width: 100%;
+  width: auto;
+  margin: 0;
+  text-align: left;
+  border: none;
+}
+/* .navigation {
```

Comment:
Is there any reason this is commented out? It would be cleaner to remove it if it's not needed